astropy/table/tests/test_table.py

```diff
@@ -144,6 +144,19 @@ def test_set_new_col_new_table_quantity(self, table_types):
         assert np.all(t['bb'] == 3)
         assert t['bb'].unit == u.m
 
+    def test_get_col_table_func_quantity(self, table_types):
```

Comment:
This test is OK in principle, but given that there is no other test of `.quantity` in this file, I think it would be more logical to replace it with a test that works directly on the `Column` instance, by adding something to `test_columns` (just after `test_to_quantity`, or perhaps as part of that).